ChiTech/ChiMath/SpatialDiscretization/FiniteElement/PiecewiseLinear/pwl_00_constrdestr.cc

```diff
@@ -35,6 +30,36 @@ SpatialDiscretization_PWLD::
                 << " Communicating partition neighbors.";
   ref_grid->CommunicatePartitionNeighborCells(neighbor_cells);
 
+  if (setup_flags == chi_math::finite_element::COMPUTE_UNIT_INTEGRALS)
+  {
+    const auto qorder_min = (int)chi_math::QuadratureOrder::SECOND;
```

Comment:
We should use proper C++ style casts.

Reply:
Agreed; in this case, likely I was attempting to match the convention employed elsewhere in the class. I have revised these instances.

Reply:
I am a bit lost here. What was the old code, and what did it need to be changed to?

Reply:
See 09112e3cf74d2a26e2b58d72a186a806fd4f9113.

Reply:
Ahhh, I meant for the change to be spelled out here. Just for easier referencing and for others to learn from, I am posting it here again:

The casting using `(int)` should be replaced with `static_cast<int>`.

A valid question is: Why? Short answer: it is good practice. The more general convention of when c-style casting can be acceptable needs a more intricate answer which we should pick up in a discussion on the boards.